### PR TITLE
MAISTRA-2107 Validate shell scripts with shellcheck

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Comment by jwendell** [comment]
@luksa create the prow job for it, then we can run the test on this very PR! :)

**Comment by luksa** [comment]
> /test all

Thanks! I was just typing a comment to ask you how to do this :)

**Comment by jwendell** [comment]
@luksa JFYI, this is the version of shellcheck we are using: https://github.com/maistra/test-infra/blob/main/docker/maistra-builder_2.3.Dockerfile#L22

**Comment by jwendell** [comment]
We should update it to 0.8.0

File: Makefile [hunk 1]
```diff
@@ -53,6 +53,10 @@ $(info   Building $(BUILD_TYPE) operator)
 
 export SOURCE_DIR OUT_DIR MAISTRA_BRANCH MAISTRA_VERSION VERSION COMMUNITY BUILD_TYPE
 
+FINDFILES=find . \( -path ./common-protos -o -path ./.git -o -path ./out -o -path ./.github -o -path ./licenses -o -path ./vendor \) -prune -o -type f
```
Comment (jwendell) [inline Makefile]: exclude ./tmp as well?

Comment (luksa) [reply Makefile]: Done. Thanks!

**Review by jwendell — APPROVED** [review]

**Pull request merged**